template/src/main/java/com/asyncapi/model/$$objectSchema$$.java

```diff
@@ -158,6 +158,9 @@ public class {{allName}} {
     {% endfor %}
     {% if params.disableEqualsHashCode === 'false' %}@Override
     public boolean equals(Object o) {
+    {%- if schema.properties() | length === 0 %}
```

Comment:
since you use `schema.properties() | length === 0` twice in this template, you can define one set for it, like for example in https://github.com/asyncapi/java-spring-template/blob/master/partials/KafkaConfig.java#L2